src/foldermodel.cpp

```diff
@@ -217,7 +226,14 @@ QVariant FolderModel::data(const QModelIndex& index, int role/* = Qt::DisplayRol
     }
     case FileInfoRole:
         return QVariant::fromValue(info);
+    case FileIsCutRole:
+        bool isCut = false;
+        if (folder_ && folder_->hasCutFiles()) { // most of the time false
+            isCut = item->isCut() || item->info->isCut(); // then dynamic evaluation
```

Comment:
Why is spread twice (in FileInfo and in FolderModelItem)? Because of const-ness of FileInfo. Could be casted away but should be?